packages/sui-ssr/package.json

```diff
@@ -43,6 +43,6 @@
     "ua-parser-js": "0.7.20"
   },
   "devDependencies": {
-    "@s-ui/test": "4"
+    "@s-ui/test": "4.11.0-beta.1"
```

Comment:
Temporary version until a needed `sui-test` feature is published: https://github.com/SUI-Components/sui/pull/1045